io.github.toCase.Birchflow/io.github.toCase.Birchflow.yml

```diff
@@ -0,0 +1,25 @@
+app-id: io.github.toCase.Birchflow
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+command: appBirchFlow
+finish-args:
+  - --filesystem=home
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --share=ipc
+modules:
+  - name: birchflow
+    buildsystem: cmake
+    subdir: src
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: git
+        url: https://github.com/toCase/birchflow.git
+        branch: main
+    post-install:
+      - install -Dm644 ../flatpak/io.github.toCase.Birchflow.svg /app/share/icons/hicolor/scalable/apps/io.github.toCase.Birchflow.svg
+      - install -Dm644 ../flatpak/io.github.toCase.Birchflow.desktop /app/share/applications/io.github.toCase.Birchflow.desktop
+      - install -Dm644 ../flatpak/io.github.toCase.Birchflow.metainfo.xml /app/share/metainfo/io.github.toCase.Birchflow.metainfo.xml
```

Comment:
these should be install by the cmake build system. *hint* they are NOT specific to flatpak in anyway.